packages/manager/src/components/TextField.test.tsx

```diff
@@ -1,9 +1,9 @@
+import { InputAdornment } from '@linode/ui/';
```

Comment:
Source of unit test failure

```suggestion
import { InputAdornment } from '@linode/ui';
```